packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -112,8 +112,8 @@ export async function getComponentsPath(
   if (hasRootWorkspace()) {
     try {
       const workspaceRootPath = getRootWorkspacePath();
-      const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
-
+      let username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
```

Comment:
This had to be updated due to the changes recently made to the `getUsername` method. If the `defaultUsernameOrAlias` is not an alias, then the `getUsername` method now returns undefined. In that scenario, we want to continue using the default value initially provided.

Reply:
A fun trick in cases like this is `let username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias) || defaultUsernameOrgAlias;`. Not necessary though :)

Reply:
Ooooh that’s literally exactly what I was trying to figure out how to do!! You rock I’m updating this